src/knx_facade.h

```diff
@@ -423,7 +406,7 @@ template <class P, class B> class KnxFacade : private SaveRestore
     ProgLedOffCallback _progLedOffCallback = 0;
     uint32_t _ledPinActiveOn = LOW;
     uint32_t _ledPin = LED_BUILTIN;
-    uint32_t _buttonPinInterruptOn = RISING;
+    uint32_t _buttonPinInterruptOn = CHANGE;

```

Comment:
Just get rid of that field and use CHANGE directly. Or is there a need to keep it?

Reply:
No need to keep it. I'll kill it.